docs/get-started/getting-started-alluxio.md

```diff
@@ -0,0 +1,200 @@
+go through how to set up the RAPIDS Accelerator for Apache Spark with Alluxio with
+a on premise cluster.
+
+## Prerequisites
+
+Assuming that user has successfully setup and ran the RAPIDS Accelerator with on premise
+cluster according to [this doc](getting-started-on-prem.md)
+
+This guide will deploy Alluxio on Yarn cluster with 2 NodeManagers and 1 ResourceManager,
+and describe the instructions to configure Swiftstack as Alluxio’s under storage system.
+
+Let's assume the hostnames of Yarn cluster are respectively
+
+``` json
+RM_hostname
+NM_hostname_1
+NM_hostname_2
+```
+
+## Alluxio setup
+
+It is recommending to deploy Alluxio workers in each NodeManager and Alluxio master in
+ResourceManager.
+
+1. Download the latest Alluxio version (2.4.1) **alluxio-${LATEST}-bin.tar.gz** from [alluxio website](https://www.alluxio.io/download/).
+2. Copy `alluxio-${LATEST}-bin.tar.gz` to all the NodeManagers and ResourceManager.
+3. Extract `alluxio-${LATEST}-bin.tar.gz` to the directory specified by **ALLUXIO_HOME**
+   in NodeManager and ResourceManager.
+
+   ``` shell
+   # Let's assume to extract alluxio to /opt
+   mkdir -p /opt
+   tar xvf alluxio-${LATEST}-bin.tar.gz -C /opt
+   export ALLUXIO_HOME=/opt/alluxio-${LATEST}
+   ```
+
+4. Configure alluxio.
+   - Alluxio master configuration
+
+   Add below recommending configuration in `${ALLUXIO_HOME}/conf/alluxio-site.properties`.
+
+   ``` xml
+   alluxio.master.hostname=RM_hostname
+
+   ######### worker properties #########
+   ## configure async cache manager
+   alluxio.worker.network.async.cache.manager.threads.max=64
+   alluxio.worker.network.async.cache.manager.queue.max=51200
+   alluxio.worker.tieredstore.levels=1
+   alluxio.worker.tieredstore.level0.alias=SSD
+   alluxio.worker.tieredstore.level0.dirs.mediumtype=SSD
+   alluxio.worker.tieredstore.level0.dirs.path=/YOUR_CACHE_DIR
+   alluxio.worker.tieredstore.level0.dirs.quota=/YOUR_CACHE_QUOTA
+
+   # enable domain socket
+   alluxio.worker.data.server.domain.socket.address=/YOUR_DOMAIN_SOCKET_PATH
+   alluxio.worker.data.server.domain.socket.as.uuid=true
+   ####################################
+
+   ######### User properties #########
+   ## enable circuit
+   alluxio.user.short.circuit.preferred=true
+   alluxio.user.short.circuit.enabled=true
+   ## use LocalFirstPolicy
+   alluxio.user.ufs.block.read.location.policy=alluxio.client.block.policy.LocalFirstPolicy
+   ####################################
+
+   # Running Alluxio Locally with S3
+   alluxio.underfs.s3.endpoint=/S3_ENDPOINT
+   alluxio.underfs.s3.inherit.acl=false
+   alluxio.underfs.s3.default.mode=0755
+   alluxio.underfs.s3.disable.dns.buckets=true
+   ```
+
+   For the explanation of each configuration, please refer to [Alluxio Configuration](https://docs.alluxio.io/os/user/stable/en/reference/Properties-List.html) and [Amazon AWS S3](https://docs.alluxio.io/os/user/stable/en/ufs/S3.html).
```

Comment:
I want more explanation then this as to exactly why our setup uses some of these configs.  Someone might have an existing cluster with different configs and wonder why we used this.